src/modules/geolocation/components/BaseMap.tsx

```diff
@@ -0,0 +1,50 @@
+import 'leaflet/dist/leaflet.css';
+
+import { Box } from '@mui/system';
+import L from 'leaflet';
+import markerIconRetinaUrl from 'leaflet/dist/images/marker-icon-2x.png';
+import markerIconUrl from 'leaflet/dist/images/marker-icon.png';
+import markerShadowUrl from 'leaflet/dist/images/marker-shadow.png';
+
+import { MapContainer, Marker, TileLayer } from 'react-leaflet';
+import { LatLon } from '@/modules/geolocation/types';
+
+// Fix for default marker icon issue with Vite
+L.Icon.Default.prototype.options.iconUrl = markerIconUrl;
+L.Icon.Default.prototype.options.iconRetinaUrl = markerIconRetinaUrl;
+L.Icon.Default.prototype.options.shadowUrl = markerShadowUrl;
+L.Icon.Default.imagePath = '';
+
+interface BaseMapProps {
+  coordinates: LatLon;
+  height?: number;
+  // todo: expand for LocationMap
+}
+
+const BaseMap: React.FC<BaseMapProps> = ({ coordinates, height = 230 }) => {
+  return (
+    <Box sx={{ height }}>
+      <MapContainer
+        key={JSON.stringify(coordinates)}
+        center={[coordinates.latitude, coordinates.longitude]}
+        zoom={16}
+        style={{ height }}
+        dragging={false}
+        scrollWheelZoom={false}
+        doubleClickZoom
+        touchZoom
+      >
+        <>
+          <TileLayer
+            url='https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
+            attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+          />
+          {/* todo: add Popup children for location map */}
+          <Marker position={[coordinates.latitude, coordinates.longitude]} />
```

Comment:
Marker should probably be given a title or alt text? Maybe the coordinates is good enough for now. (I think even better would be a description, like "in San Francisco on Main Street" but we probably don't want to pull in a library to do that)

Reply:
Added a basic title "client location". The enrollment location map (other pr) will have content in the tooltip, but I don't think we need that here. It's probably not helpful to read aloud the coordinates, right?

Reply:
I guess so yeah. It's too bad though, since without the coordinates in the marker description, the actual location data is info that is only available to sighted users. We can leave it for now though; we could add a ticket for a future improvement, for example when we add additional capabilities like entering an address to drop a pin, that might be a good time to bring in a library that can convert a coordinate pair into a useful verbal description.